celery/bin/multi.py

```diff
@@ -469,7 +469,7 @@ def DOWN(self):
 )
 @click.pass_context
 @handle_preload_options
-def multi(ctx):
+def multi(ctx, **kwargs):
```

Comment:
This won't do anything but accept additional arguments.
You aren't passing it anywhere 🤷 .

Can you provide a test case where this works as intended?